src/containers/AdminDashboard/AdminDashboardContent.tsx

```diff
@@ -61,13 +69,16 @@ function CommentsEditor(
   );
 }
 
-interface IchangeHomepageProps {
-  homeContent: Ibook, dispatch: Dispatch<AnyAction>
-}
-function ChangeHomepage(props: IchangeHomepageProps): JSX.Element {
-  const { homeContent, dispatch } = props;
-  const [title, setTitle] = useState(homeContent.title);
-  const [comments, setComments] = useState(homeContent.comments);
+// interface IchangeHomepageProps {
+//   // homeContent: Ibook, dispatch: Dispatch<AnyAction>
```

Comment:
@kathleenahall please delete commented out code before asking for code reviews, thanks